aeron-annotations/src/main/java/io/aeron/config/ConfigInfo.java

```diff
@@ -47,9 +47,9 @@ public ConfigInfo(final String id)
     public String defaultDescription;
     public String defaultFieldName;
     public String defaultClassName;
-    public Object defaultValue;
```

Comment:
This feels dirty... but... java 21 gets upset about serializing 'Object'.  After deserialization, I was already using 'instanceof' to figure out what I was looking at... so switching to Serializable still seems to work fine.  But it still feels dirty.